internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -167,7 +177,15 @@ func (q amsQuotaService) ValidateBillingAccount(organisationId string, instanceT
 	return nil
 }
 
-func (q amsQuotaService) CheckIfQuotaIsDefinedForInstanceType(username string, externalId string, instanceType types.KafkaInstanceType) (bool, *errors.ServiceError) {
+func (q amsQuotaService) CheckIfQuotaIsDefinedForInstanceType(username string, externalId string, instanceType types.KafkaInstanceType, billingModel services.KafkaBillingModel) (bool, *errors.ServiceError) {
+	//
```

Comment:
This is commented. Is this intentional? If so, we can just add it in another PR or at least add some context. Also it seems that the `AmsBillingModel` interface is only used for this commented code so we should also comment it or delete it

Reply:
Yes, it was commented on purpose. This PR is about changes to the QUOTA-LIST management, but it required some changes to the shared `QuotaService` interface too, so I added a comment to explain what was the intended usage when implementing the AMS Quota manager too.

I'm going to assign that task to myself, so this comment doesn't make sense anymore: removing.